<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="libStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="homePage">
    <widget class="QLabel" name="systemHeader">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>711</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to the Library System!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchForBook">
    <widget class="QLabel" name="searchPageHeader">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>711</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Search the database for a book.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>270</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>210</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Please enter the ID number or title of the book.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputLabel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>340</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="enterID">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>230</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="outputLabel_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>400</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="outputLabel_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>460</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="addNewBook">
    <widget class="QLabel" name="addBookHeader">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>711</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Add a new book to the database.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="desclab">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Please enter the name, year of publication, and author of the book you are adding.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>420</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="namelab">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>249</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="yearlab">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>298</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>YEAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="authorlab">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AUTHOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="hiddenLab">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>530</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="addInputGroup">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>230</y>
       <width>391</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>                                                                                                 </string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="authorEntry">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="yearEntry">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEntry">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="viewBook"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
